bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ParameterGroup.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.config.core;
+
+/**
+ * The {@link ParameterGroup} specifies information about parameter groups.
+ * A parameter group is used to group a number of parameters together so they can
+ * be displayed together in the UI (eg in a single tab).
+ * <p>
+ * A {@link ConfigDescriptionParameter} instance must also contain the group name. It should be permissible to use the
+ * group name in the {@link ConfigDesctiptionParameter} without supplying a corresponding {@link ParameterGroup} - in
+ * this way the UI can group the parameters together, but doesn't have the group information.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class ParameterGroup {

```

Comment:
No problem with the above changes.
The reason I went with ParameterGroup is because there is also ParameterOption - I'm happy to change though (it was a tossup between the two anyway).